client/src/pages/Review/Review.tsx

```diff
@@ -1,39 +1,87 @@
+import { useState } from "react";
+import { useParams, Link } from "react-router-dom";
 import Button from "../../components/Button";
+import { classnames } from "../../Utilities";
+
+const courses = ["cpsc-413", "seng-513", "cpsc-481", "cpsc-449", "cpsc-457"];
 
 const Review = () => {
+  const [selectedTab, setSelectedTab] = useState(0);
+  const { courseId } = useParams();
+  const format = (courseId: string | undefined) => courseId?.replace("-", " ").toUpperCase();
+
   return (
-    <div className="grid grid-cols-5">
-      {/* Left column: navigation bar */}
-      <nav className="col-span-1 bg-gray-800 p-4 flex flex-col">
-        {/* Navigation content */}
-        <Button variant="filled">CPSC 100</Button>
-        <Button variant="filled">CPSC 100</Button>
-        <Button variant="filled">CPSC 100</Button>
-        <Button variant="filled">CPSC 100</Button>
-        <Button variant="filled">CPSC 100</Button>
+    <div className="flex h-screen">
+      <nav
+        className={classnames(
+          "w-full", "md:w-64", "bg-gray-600", "h-full", "fixed", "left-0", "top-0", "flex", "flex-col", "p-4", "items-center", "z-0", !courseId && "z-20"
+        )}
+      >
+        <div className="flex flex-col h-full">
+          {courses.map((course) => (
+            <Link to={`/review/${course}`} key={course}>
+              <Button variant={courseId === course ? "filled" : "tonal"}>
+                {format(course)}
+              </Button>
+            </Link>
+          ))}
+        </div>
       </nav>
+      <div className="flex flex-col w-full ml-0 md:ml-64 z-10">
```

Comment:
The main container must have a maximum width to adapt wide screen, and the container should be center-aligned

<img width="450" alt="Screenshot 2023-02-03 at 7 48 37 PM" src="https://user-images.githubusercontent.com/20573623/216742607-d9b411cc-48b8-4ed2-a9d5-9b9601bffb8b.png">

<img width="900" alt="Screenshot 2023-02-03 at 7 48 45 PM" src="https://user-images.githubusercontent.com/20573623/216742619-c0f1a5ba-bb21-4bde-999e-e65db2b0eb0c.png">